right-for-life_front/src/components/ArticleImageGallery/ArticleImageGallery.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import ImageGallery from 'react-image-gallery';
+
+import './style.css';
+
+export const ArticleImageGallery = ({images}) => {
+  images = images.map(image => ({original: image, thumbnail: image}));
+
+  return <ImageGallery items={images} showPlayButton={false}/>
```

Comment:
[optional] Maybe it’s worth remaking the gallery control buttons to fit the site design.